tests/conftest.py

```diff
@@ -1197,3 +1272,38 @@ def nodes():
     factory = platform_nodes.PlatformNodesFactory()
     nodes = factory.get_nodes_platform()
     return nodes
+
+
+@pytest.fixture(scope='session')
+def default_storageclasses(request, teardown_factory_session):
+    """
+    Returns dictionary with storageclasses. Keys represent reclaim policy of
+    storageclass. There are two storageclasses for each key. First is RBD based
+    and the second one is CephFS based. Storageclasses with Retain Reclaim
+    Policy are created from default storageclasses.
```

Comment:
If possible, we can make it:
 Storageclasses with ReclaimPolicy: Retain  are created from default storageclasses.

Reply:
In the pull request I I clone default storageclasses and change the name and reclaim policy to Retain in order to get a storageclass with Retain reclaim policy.